.github/workflows/build.yml

```diff
@@ -0,0 +1,41 @@
+name: CI QA
+
+on:
+  push:
+  # pull_request:
+  #   branches:
+  #     - master
+
+jobs:
+  qa:
+    runs-on: ubuntu-20.04
```

Comment:
Better to use 22.04?

Reply:
I'll check that, I had problems with py36 on 22.04